ical/event.py

```diff
@@ -109,7 +111,7 @@ class Event(ComponentModel):
     duration: Optional[datetime.timedelta] = None
     """The duration of the event as an alternative to an explicit end date/time."""
 
-    summary: str = ""
+    summary: Optional[str] = None
```

Comment:
Can we avoid making breaking typing changes? I think i did this to avoid needing to do `None` checks for some of the most common default fields.

Reply:
OK, fine, not the end of the world.